templates/patient_extension.rb.erb

```diff
@@ -0,0 +1,137 @@
+  field :ethnicity, type: Code
```

Comment:
(Posting for visibility) What was the motivation behind templating the patient?

Reply:
QDM actually already has a patient datatype. So we build the corresponding Patient model from what is in the modelinfo file, and then use this extension to add the additional functionality we need for Bonnie and Cypress. Using an ERB allows us to 1: separate out this code in a clean and easy to modify manner, and 2: allows us to at any point in the future make this extension more dynamic if need be.